modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTxDistributedTestSingleNodeCollocated.java

```diff
@@ -46,15 +44,6 @@ protected boolean startClient() {
     @Override
     public void before() throws Exception {
         super.before();
-
```

Comment:
Could you explain why these assertions were useful and why we shouldn't have the analog now?

Reply:
It's explained in [the ticket's description](https://issues.apache.org/jira/browse/IGNITE-22774)